sql-cli/sql_cli/project.py

```diff
@@ -0,0 +1,18 @@
+import os
+import shutil
+from pathlib import Path
+
+BASE_SOURCE_DIR = Path(Path(os.path.realpath(__file__)).parent.parent, "include/base/")
```

Comment:
```suggestion
BASE_SOURCE_DIR = Path(__file__).parent.parent / "include/base"
```
doesn't this work as well?